lib/src/command/add.dart

```diff
@@ -68,18 +68,18 @@ For example:
   * Add multiple dependencies:
     `$topLevelProgram pub add foo dev:bar`
   * Add a path dependency:
-    `$topLevelProgram pub add 'foo:{"path":"../foo"}'`
+    `$topLevelProgram pub add foo:{"path":"../foo"}`
```

Comment:
I think it is better to leave these quotes, otherwise bash will split the input if someone inserts a space.

Reply:
Reverted. The previous pattern is now acceptable.